arch/cortex-m3/src/mpu.rs

```diff
@@ -364,22 +364,19 @@ impl kernel::mpu::MPU for MPU {
     type MpuConfig = CortexMConfig;
 
     fn enable_mpu(&self) {
-        let regs = &*self.registers;
-
```

Comment:
If we don't need this line of code with the obscure `&*` everywhere, then I definitely want it gone.